docs/specs/metadata.yml

```diff
@@ -95,6 +97,8 @@ inputs:
     product: Win
     files: "**/*"
     baseUrl: https://docs.com/sccm
+    routes:
```

Comment:
there are two concerns:
- User need combine two config "baseUrl" and "routes" to get the finally site url
- Routes expose something like `internal mapping` out to end user, it doesn't map to output path or site path

Reply:
actually the `routes` in this change is more like `sourceBasePath`.

Reply:
we can provide `sourceBasePath` + `baseUrl` experiences to docs user and `routes` to community users